src/AbstractEvent.php

```diff
@@ -19,7 +19,7 @@
     protected $emitter;
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
```

Comment:
Changing this from an inline tag to a normal tag changes it's behaviour so that it no longer inherits what it was inheriting before. See https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#6-inheritance.